engine/lite/embeddedDeploymentManager/src/test/scala/pl/touk/nussknacker/streaming/embedded/EmbeddedDeploymentManagerRestartTest.scala

```diff
@@ -22,8 +22,6 @@ class EmbeddedDeploymentManagerRestartTest extends BaseEmbeddedDeploymentManager
       fixture.deployScenario(scenario)
     }
 
-    manager.findJobStatus(name).futureValue.map(_.status) shouldBe Some(SimpleStateStatus.Running)
-
     kafkaZookeeperServer.kafkaServer.shutdown()
```

Comment:
wouldn't it now be clearer if this shutdown and await would be done after Restarting check - right befeore line 34

Reply:
I'm not sure what you mean by that, the shutdown needs to complete for this assertion to pass `jobStatus.map(_.status) shouldBe Some(EmbeddedStateStatus.Restarting)`

Reply:
hmm I think I checked it last week and this assertion passed regardless of shutdown placement

Reply:
Could be that because of the timeout which was set too low, the deployment went into Restarting status regardless of the shutdown... should be better now.